[fix] object request url to v2

Achoobert opened the pull request:
https://github.com/digi-serve/sdc-app/issues/34

Commit:
Add Service "GetResetPasswordUrl"

Commit:
Fix data processor on ABProcessTaskServiceGetResetPasswordUrlCore.js

Hiro-Nakamura committed:
[add] make sure ABProcessParticipant defines a .key

Commit:
Add the value "url" on ABProcessTaskServiceGetResetPasswordUrlCore.js

Commit:
Fix returned processDataField's values

Commit:
Remove url param

Commit:
Merge branch 'v2' of https://github.com/digi-serve/appbuilder_class_core into NewServiceGetResetPasswordUrl

Hiro-Nakamura committed:
[fix] initial error handling of missing object in ABQLSetPluck

jinzsom committed:
Merge pull request #72 from digi-serve/NewServiceGetResetPasswordUrl

Service Reset Password URL

nh758 committed:
fix error when `this.uuid()` not defined (#74)

* fix error when `this.uuid()` not defined

* switch to use this.AB

Hiro-Nakamura committed:
[add] expose ABMLClass in our ABFactory

Hiro-Nakamura committed:
expose ABViewFormComponent in ABFactory / Refactor ABViewFormConnect to not store settings in .objectWorkspace

nh758 committed:
fix list validation

nh758 committed:
Add support for `{PK}` tag in templates. (#75)

* Add support for `{uuid}` tag in templates.

* switch to use `.PK()`

wongpratan committed:
*Fix this_object option in FilterComplex - https://github.com/digi-serve/ab_platform_web/issues/40

wongpratan committed:
*Fix .isComplete function of FilterComplex - https://github.com/digi-serve/ab_platform_web/issues/40

Hiro-Nakamura committed:
[add] interface designer modifications

Hiro-Nakamura committed:
[add] updated english labels

Hiro-Nakamura committed:
[fix] notify() routine will not break if missing an {info} object

wongpratan committed:
*Fix process field options in ComplexFilter - https://github.com/digi-serve/ab_platform_web/issues/40

wongpratan committed:
Merge pull request #77 from digi-serve/pp/fixComplexFilter

*Fix this_object option in FilterComplex

wongpratan committed:
*Fix pull relation data - https://github.com/digi-serve/kitchensink_app/issues/31

jinzsom committed:
Add the new user task "External"

jinzsom committed:
Merge pull request #78 from digi-serve/guy/addUserTaskExternal

Add the new user task "External"

wongpratan committed:
+Add No Item Reported label - https://github.com/digi-serve/cars_app/issues/51

wongpratan committed:
*Fix populate data of dc when parent dc cursor is changed - https://github.com/digi-serve/well_app/issues/16

wongpratan committed:
+Allow set a fallback label - https://github.com/digi-serve/appbuilder_class_core/pull/80#discussion_r911598533

wongpratan committed:
Merge pull request #79 from digi-serve/fixDcCursor

Fix populate DC when parent DC cursor is changed

wongpratan committed:
Merge pull request #80 from digi-serve/NoneOptionV2

Add No Item Reported label

wongpratan committed:
+Include Detail widget file to ABFactory - https://github.com/digi-serve/plugin_ABDesigner/issues/42

wongpratan committed:
*Use ABViewDetailItemCore - https://github.com/digi-serve/plugin_ABDesigner/issues/42

wongpratan committed:
*Use ?? instead of || - https://github.com/digi-serve/appbuilder_class_core/pull/81#discussion_r919627764

wongpratan committed:
*Use let instead of var - https://github.com/digi-serve/appbuilder_class_core/pull/81#discussion_r919631437

wongpratan committed:
Merge pull request #81 from digi-serve/detailWidget

Detail widget

wongpratan committed:
+Migrate DOCX builder widget - https://github.com/digi-serve/plugin_ABDesigner/issues/47

wongpratan committed:
+Bring it back - https://github.com/digi-serve/appbuilder_class_core/pull/82#discussion_r924258216

wongpratan committed:
*Move .letUserDownload to ab_platform_web - https://github.com/digi-serve/appbuilder_class_core/pull/82#discussion_r924266592

wongpratan committed:
Merge pull request #82 from digi-serve/docxWidget

Migrate DOCX builder widget

jinzsom committed:
Fix multiselect

wongpratan committed:
+Include dataview widget - https://github.com/digi-serve/plugin_ABDesigner/issues/46

wongpratan committed:
+Override .parentDetailComponent function - https://github.com/digi-serve/plugin_ABDesigner/issues/46

jinzsom committed:
Merge pull request #83 from digi-serve/guy/fixMultiselect

Fix multiselect

wongpratan committed:
Merge pull request #86 from digi-serve/dataviewWidget

Migrate Dataview widget

wongpratan committed:
*[workaround] Comment include ABProcessTaskUserExternal file

nh758 committed:
Add workflow to test and update dependent repos (#84)

* Create test-and-update.yml

* fix syntax

* fix webpack command

* fix checkout path

* fix names

* checkout develop branches for services

nh758 committed:
fix checkout command

wongpratan committed:
*Fix filter options to custom index - https://team.digiserve.org/main/pl/y13jsoy4rb8idbhbciygugxwae

wongpratan committed:
Merge pull request #87 from digi-serve/pp/fixCustomFilter

*Fix filter options to custom index

wongpratan committed:
Merge branch 'v2' of github.com:digi-serve/appbuilder_class_core into ABDesigner

wongpratan committed:
+Include ABProcessTaskUserExternal

jinzsom committed:
Fix "useAsLabel" value

jinzsom committed:
Merge pull request #89 from digi-serve/guy/FixUseAsLabel

Fix use as label

jinzsom committed:
Fix UI label and default key

jinzsom committed:
Merge pull request #90 from digi-serve/guy/MigrateWigetLabel

Fix UI label and default key

Commit:
migrate v1 changes in to v2 for filteredClones of data collections

nh758 committed:
Merge pull request #91 from digi-serve/jd/fix_filteredClone_rules

migrate v1 changes in to v2 for filteredClones of data collections

Commit:
migrate connected field filter rules and other small fixes

Commit:
more tweaks to filter complex

Commit:
more tweaks

Commit:
remove unused variables

Commit:
remove unused variable

joshAppDev committed:
Changed single-quote strings to double-quote strings.
ab_service_appbuilder/handlers/labelMissing.js is trying to parse this file as JSON

jinzsom committed:
Migrate widget comment

jinzsom committed:
Merge pull request #97 from digi-serve/guy/MigrateWidgetComment

Migrate widget comment

wongpratan committed:
*Migrate conditional container widget

wongpratan committed:
Merge pull request #98 from digi-serve/pp/conditionalContainer

*Migrate conditional container widget

jinzsom committed:
Fix bug isUuid

jinzsom committed:
Merge pull request #99 from digi-serve/guy/FixBug

Fix bug isUuid

wongpratan committed:
*Use common functions instead of third party in core

wongpratan committed:
Merge branch 'v2' of github.com:digi-serve/appbuilder_class_core into ABDesigner

Commit:
change conditionals from || to ??

Commit:
Merge commit '377a4dfcff8dcbe38d7526e26a415f41da58b7d7' into jd/filterComplexFixes

* commit '377a4dfcff8dcbe38d7526e26a415f41da58b7d7':
  *Use common functions instead of third party in core
  Fix bug isUuid
  Changed single-quote strings to double-quote strings. ab_service_appbuilder/handlers/labelMissing.js is trying to parse this file as JSON

jamesduncan committed:
Merge pull request #96 from digi-serve/jd/filterComplexFixes

Jd/filter complex fixes

Commit:
add labels

wongpratan committed:
Merge branch 'v2' of github.com:digi-serve/appbuilder_class_core into ABDesigner

wongpratan committed:
Revert "*Apply real time update to support user scope"

This reverts commit ef8127028ed4c56f61d51d89f3ae6753c713e989.

wongpratan committed:
*Apply real time update to support user scope

wongpratan committed:
Merge pull request #100 from digi-serve/pp/realTimeUpdate

*Apply real time update to support user scope

wongpratan committed:
*[hotfix]undefined task

wongpratan committed:
Merge pull request #101 from digi-serve/pp/fixUndefinedTask

Fix undefined task error

nh758 committed:
save AB factory if we have it

Commit:
clone dc before returning it

jamesduncan committed:
Merge pull request #103 from digi-serve/jd/clone_cursor_fix

clone dc before returning it

wongpratan committed:
*Fix pull queries in FilterComplex - https://github.com/digi-serve/ns_app/issues/105

wongpratan committed:
*Clean code - https://github.com/digi-serve/appbuilder_class_core/pull/104#discussion_r978238966

wongpratan committed:
Merge pull request #104 from digi-serve/pp/fixQueryInComplexFilter

*Fix pull queries in FilterComplex

nh758 committed:
add widget to filter datacollection by a connected record

wongpratan committed:
*Migrate Form widget

wongpratan committed:
*Fix reference

wongpratan committed:
Merge branch 'v2' of github.com:digi-serve/appbuilder_class_core into ABDesigner

wongpratan committed:
*Merge from #ABDesigner

wongpratan committed:
[hotfix] generate AGE value to Docx CARS reports - https://github.com/digi-serve/cars_app/issues/85

wongpratan committed:
Merge branch 'v2' of github.com:digi-serve/appbuilder_class_core into ABDesigner

wongpratan committed:
Merge branch 'ABDesigner' of github.com:digi-serve/appbuilder_class_core into pp/formWidget

wongpratan committed:
Merge pull request #106 from digi-serve/pp/formWidget

Migrate Form widget

jinzsom committed:
Migrate the widget "Chart"

jinzsom committed:
Merge pull request #107 from digi-serve/guy/MigrateWidgetChart

Migrate widget chart

nh758 committed:
set calculate field isFilterable based on settings.formula

Achoobert committed:
[add] email field definitions

Achoobert committed:
[add] check if attribute is null

Achoobert committed:
Merge pull request #109 from digi-serve/is/emailTask

[add] email field definitions

wongpratan committed:
*Use .ABFactory - https://github.com/digi-serve/ns_app/issues/137#issuecomment-1272027905

wongpratan committed:
*Use .objectByID function instead - https://github.com/digi-serve/appbuilder_class_core/pull/110#pullrequestreview-1135628179

wongpratan committed:
Merge pull request #110 from digi-serve/participantUserField

*Use .AB instead of .application

wongpratan committed:
*Set Auto index field to be sortable - https://github.com/digi-serve/ns_app/issues/128

wongpratan committed:
*Fix is_empty and is_not_empty - https://github.com/digi-serve/ns_app/issues/147

Commit:
Merge commit 'bb9b4c24f3a4e5d9005aafd2bb16d54ea9f15a6d' into ABDesigner

wongpratan committed:
Merge pull request #112 from digi-serve/fix/isEmpty

*Fix is_empty and is_not_empty

wongpratan committed:
Merge pull request #111 from digi-serve/fix/autoIndexSortable

Set auto index field to be sortable

nh758 committed:
allow sort by date

Commit:
Merge commit '97e438ee09a1a468269181dff53fff54284f69f9' into ABDesigner

wongpratan committed:
*Fix date picker in FilterComplex - https://github.com/digi-serve/ab_platform_web/issues/183

Commit:
[add] ABProcess now registers warnings

wongpratan committed:
*Fix undefined - https://github.com/digi-serve/appbuilder_class_core/pull/114

wongpratan committed:
Merge pull request #114 from digi-serve/fix/dateFilter

*Fix date picker in FilterComplex

wongpratan committed:
+Add show/hide the save button in filterComplex - https://team.digiserve.org/main/pl/s781fncgyp8pim1uhyswxis11c

lgtm-migrator committed:
Add CodeQL workflow for GitHub code scanning

Commit:
[add] ABApplication now returns live process instances.

wongpratan committed:
Merge branch 'v2' of github.com:digi-serve/appbuilder_class_core into ABDesigner

Commit:
Fix populating QLOP

jinzsom committed:
var -> const or let, fix eslint error and format prettier

Commit:
[add] prevent SubProcess from loosing unsaved changes

Commit:
Merge commit '8cbeebd3d616acefc2be9fb95b623f06ee76e705' into jh/ProcessWarnings

Commit:
[add] check for a valid parent from connections

wongpratan committed:
Merge pull request #115 from digi-serve/complexFilter/formulaField

+Add show/hide the save button in filterComplex

Commit:
[add] access the SiteLanguage object from the ABFactory

Hiro-Nakamura committed:
Merge pull request #119 from digi-serve/jh/FixAPICalls

[add] access the SiteLanguage object from the ABFactory

Hiro-Nakamura committed:
Merge pull request #118 from digi-serve/jh/ProcessWarnings

Jh/process warnings

Commit:
Merge branch 'v2' into ABDesigner

Commit:
[add] updated switcheroo related labels

jinzsom committed:
Merge branch 'v2' into guy/FixProcessManager

jinzsom committed:
Fix pluck param "field" -> "fieldID"

jinzsom committed:
Fix pluck nextQLOps

jinzsom committed:
var -> const, let

jinzsom committed:
Fix process QueryTask to process by field type with same type of value

nh758 committed:
Add populate options matching model.find cond (#120)

* add populate options matching model.find cond

* fix JSON.parse not needed and support when .populate == false

* fix reading .preventPopulate == "1"

jinzsom committed:
Merge branch 'v2' into guy/FixProcessManager

jinzsom committed:
var -> const, let and some || -> ?? and some == -> === and some != -> !== and prettier

Commit:
Merge commit 'faeefbb7b7ffc2baa0c233c9497131c472781221' into ABDesigner

jinzsom committed:
Fix select label

Commit:
new param to signify when calculated field is part of a grouped row

this gives us the ability to return only the number in order for us to do sums in the grouped row

jamesduncan committed:
Merge pull request #121 from digi-serve/jd/groupedFormulas

new param to signify when calculated field is part of a grouped row

wongpratan committed:
*Fix the settings of Grid widget - https://github.com/digi-serve/plugin_ABDesigner/issues/101

wongpratan committed:
Merge pull request #122 from digi-serve/fix/gridSettings

*Fix the settings of Grid widget

jinzsom committed:
Merge pull request #117 from digi-serve/guy/FixProcessManager

Fix Process Manager

Commit:
[wip] Added Warnings for GatewayExclusive task.

wongpratan committed:
Merge pull request #123 from digi-serve/ABDesigner

Merge new Interface & Process builders

Commit:
[wip] added error checking for email task

wongpratan committed:
*Fix display all data in the grid widget when update parent DC's cursor - https://github.com/digi-serve/ns_app/issues/191

wongpratan committed:
*Fix display data items in the grid widget - https://github.com/digi-serve/ns_app/issues/191

wongpratan committed:
*Fix [test cases of] display data items in the grid widget - https://github.com/digi-serve/ns_app/issues/191

wongpratan committed:
*Update timeout number - https://github.com/digi-serve/appbuilder_class_core/commit/f50fe8a71e4956a57c2ab55156cba284bcdde82d

wongpratan committed:
Merge pull request #125 from digi-serve/fix/grid

*Fix display all data in the grid widget when update parent DC's cursor

Commit:
[wip]  refactoring warning displays and included InsertRecord and Query tasks

Commit:
Merge commit 'b145d0999d2f7c18a4c8ba8a30a6ffff1f381487' into jh/processTaskWarnings

# Conflicts:
#	process/tasks/ABProcessTaskServiceQueryCore.js

nh758 committed:
Add default sort to object, dc, queries lists (#126)

* add default sort to object, dc, queries list

* fix doc typo

wongpratan committed:
*Fix toDate to rules.toDate - https://github.com/digi-serve/ab_service_process_manager/issues/39

wongpratan committed:
Merge pull request #127 from digi-serve/fix/processDate

*Fix toDate to rules.toDate

Commit:
Merge commit '94345086a1021e2fd3b4305ebd348eea80bcd755' into jh/processTaskWarnings

jinzsom committed:
Add param name

jinzsom committed:
Merge branch 'v2' into guy/RefactorViewComponentIDs

jinzsom committed:
Merge pull request #128 from digi-serve/guy/RefactorViewComponentIDs

Refactor view ids.component

nh758 committed:
fix cy tests in workflow (#129)

wongpratan committed:
[hotfix] .id of query value options of Sub Process task - https://github.com/digi-serve/appbuilder_class_core/blob/master/process/tasks/ABProcessTaskSubProcessCore.js#L128

wongpratan committed:
[hotfix] Pull value in sub process task - https://github.com/digi-serve/appbuilder_class_core/blob/master/process/tasks/ABProcessTaskSubProcessCore.js#L179

Commit:
[add] additional warnings

Commit:
Merge commit 'bbb6b289d52fc0bdefa51cb8bb6799b321b72845' into jh/processTaskWarnings

# Conflicts:
#	process/tasks/ABProcessTaskSubProcessCore.js

Commit:
[add] refactor the .warnings() and .warningEval() interface for Processes

Commit:
create new data filter widget

Hiro-Nakamura committed:
Merge pull request #131 from digi-serve/jh/processTaskWarnings

Jh/process task warnings

jamesduncan committed:
Merge pull request #132 from digi-serve/jd/data_filter_widget

create new data filter widget

Commit:
[add] initial refactor of ABObject and respective datafields

wongpratan committed:
*[hotfix] labels of widgets to display in Add Widget popup

Achoobert committed:
[add] keep version number as created elsewhere

https://github.com/digi-serve/plugin_ABDesigner/issues/108

Commit:
[add] refactoring ABObjectQuery warning interface.

Commit:
Merge branch 'v2' into jh/refactorWarnings

Commit:
add json form view

Achoobert committed:
[fix] move versionData to it's own object in `json`

Also change it to an Object format rather than just a string

Additionally, if data does not exist: create an "initial" version with 'initial' changelog

Hiro-Nakamura committed:
Merge pull request #133 from digi-serve/jh/refactorWarnings

Jh/refactor warnings

wongpratan committed:
*[hotfix] undefined error

Commit:
fix error where core was looking for document

wongpratan committed:
*[hotfix] undefined error

wongpratan committed:
*[hotfix] undefined error

Achoobert committed:
match style expected in designer

Commit:
[wip] initial view warnings display

Commit:
Merge commit 'f59b36ef08e94fffb9dd77c974e15a34d2bba0f0' into jd/filtercomplex_dom_error_fix

* commit 'f59b36ef08e94fffb9dd77c974e15a34d2bba0f0':
  *[hotfix] undefined error
  *[hotfix] undefined error

jinzsom committed:
Fix FilterComplex conditions to only cover scopeQuery for ProcessUserTask

jinzsom committed:
Merge branch 'v2' into guy/AddScopeQuery

nh758 committed:
fix notify info expected format

nh758 committed:
use branch v2

Commit:
Merge branch 'v2' into jd/view_form_json

* v2:
  use branch v2
  Add CodeQL workflow for GitHub code scanning
  fix notify info expected format
  *[hotfix] undefined error
  *[hotfix] undefined error
  *[hotfix] undefined error
  [add] refactoring ABObjectQuery warning interface.
  [add] initial refactor of ABObject and respective datafields

Commit:
Merge branch 'jd/filtercomplex_dom_error_fix' into jd/view_form_json

* jd/filtercomplex_dom_error_fix:
  fix error where core was looking for document

jinzsom committed:
Merge branch 'v2' into guy/AddScopeQuery

jinzsom committed:
Merge pull request #136 from digi-serve/guy/AddScopeQuery

Add scopeQuery for UserTask

jamesduncan committed:
Merge pull request #140 from digi-serve/jd/view_form_json

add json form view

Commit:
[wip] initial ABViews

wongpratan committed:
*Fix undefined functions of the sub task - https://github.com/digi-serve/plugin_ABDesigner/issues/162

wongpratan committed:
Merge pull request #141 from digi-serve/fix/subTask

*Fix undefined functions of the sub task

Achoobert committed:
[fix] data structure

https://github.com/digi-serve/plugin_ABDesigner/issues/153

Achoobert committed:
fix default datastructure

Don't set the versionNumber at the application level

Achoobert committed:
fix: setting data in wrong order

Achoobert committed:
[fix] requested refactor

Code refactor done, duplicates we bet,
Data setup and versionNumber unset.
Oh dear me, please do tell,
What's the purpose of obj to string spell?

Achoobert committed:
[fix] stop language updating deleting the versionData

Commit:
columnName is not unique across multiple objects we need to just use id

jamesduncan committed:
Merge pull request #142 from digi-serve/hotfix_query_ids

columnName is not unique across multiple objects we need to just use id

Commit:
DOMParser is not available switch to replace

jamesduncan committed:
Merge pull request #143 from digi-serve/jd/fix_removeHtmlTags

DOMParser is not available switch to replace

Achoobert committed:
[fix] keep versionData in ABApplicationCore and extract before toObj() 

In lower ABDefinition and ABMLObject methods,
VersionData brought many bugs and mods,
Now in ABApplicationCore.toObj() it shall stay,
Just grab the value out before .toObj() takes it away.

jamesduncan committed:
Check field exists before setting result

When clearing a cursor we the rowData is null, thus all checks on rowData for values will fail.

jamesduncan committed:
Merge pull request #144 from digi-serve/hotfix_datavalue_bug

Check field exists before setting result

wongpratan committed:
*Fix pull the start task in insert task

jamesduncan committed:
Fix error on format when we pass null for rowData

jamesduncan committed:
Merge pull request #146 from digi-serve/hotfix_formt_error

Fix error on format when we pass null for rowData

Achoobert committed:
Merge pull request #137 from digi-serve/is/verion.tracking.experimental

Enable saving/exporting changelog data

nh758 committed:
Hotfix push to undefined

wongpratan committed:
Merge pull request #145 from digi-serve/fix/pullStartTask

*Fix pull the start task in insert task

Achoobert committed:
wip

Commit:
[wip] cleaning up the Interface UI

Commit:
Merge commit '9186bf1a2178ffef191b71122932fa65b42b6b3d' into jh/InterfaceWarnings

Commit:
[add] updated english interface labels

wongpratan committed:
*Fix Update Connected Record - https://github.com/digi-serve/ns_app/issues/223

wongpratan committed:
Merge pull request #148 from digi-serve/fix/updatedConnctedRecord

*Fix Update Connected Record

Achoobert committed:
[add] this.results, include parent connections in returned connections

NOTE this may cause performance issues
this.results needs to be defined in insert process

Achoobert committed:
remove logging

wongpratan committed:
+Add the follow cursor setting to DC - https://github.com/digi-serve/ab_platform_web/issues/279

Achoobert committed:
[add] inserted record to query list

Achoobert committed:
[add] special filter for insert  tasks

Achoobert committed:
[fix] correct a comment

Achoobert committed:
Merge pull request #149 from digi-serve/is/processTaskFromParent

access process data from insert task from inside subtask

wongpratan committed:
*Apply how to pull and set cursor of the data collection who set follow cursor - https://github.com/digi-serve/ab_platform_web/issues/279

Hiro-Nakamura committed:
Merge pull request #147 from digi-serve/jh/InterfaceWarnings

Jh/interface warnings

wongpratan committed:
*Apply logic of DC who follow cursor in real time events - https://github.com/digi-serve/ab_platform_web/issues/279

wongpratan committed:
Merge pull request #150 from digi-serve/pp/followCursor

Filter Data Collection by another Data Collections Cursor

Achoobert committed:
[fix] concat not combine _connections

fixing that the....
"the subprocess used to continually add connection info to it's internal copy with no way for us to remove those connections.  So say we add a Sub Process,  it copies the parent's connection list into it's own, then the parent changes it's connections (we modify the graph somehow).  The Sub Process wont REMOVE those deleted connections.  It will still think they are connected and since they were copied into it's own this._connections they are now persisted."

Achoobert committed:
Revert "[fix] object request url to v2"

This reverts commit cf8f313cb9f6e5157d77e6115bfd88f5d4e52174.

Achoobert committed:
[fix] object request url to v2

Achoobert committed:
[add] field as object, to appease RowFilterCore

Achoobert committed:
Replace key == subprocess

Achoobert committed:
Merge pull request #152 from digi-serve/is/hotfixProcessConnections

Is/hotfix process connections

wongpratan committed:
+Add .fieldsAddFiltersCustom function to FilterComplexCore.js https://github.com/digi-serve/appbuilder_class_core/blob/master/FilterComplexCore.js#L1066 https://github.com/digi-serve/ns_app/issues/232

wongpratan committed:
Merge pull request #153 from digi-serve/fix/customFilterComplex

+Add .fieldsAddFiltersCustom function to FilterComplexCore

Achoobert committed:
[fix] treat user like connect object

https://github.com/digi-serve/ab_service_process_manager/issues/54

Hiro-Nakamura commented:
So, it seems like you are trying to get the SDC app to connect with `v2`, is that right?

You have found the right place where the urls are generated.  However I would recommend not editing them here.  Instead I would suggest you take the `core#v2` versions of the urls and add them to the `sdc/platform/ABObject.js` file.  Putting them there will overwrite these definitions in the `core#master/ABObject` and the SDC will be using the new urls.

The challenge here will be that there are some other changes in `v2` that may not be obvious and will have an impact on this `v1` code.  But doing this one fix will probably get us 90% there.

jamesduncan committed:
Handle null rowData after unset cusor

When we set a cursor to null to unset we also fire a series of updates to the UI. If we do not handle properly a null value here the code errors and the UI is not updated.

jamesduncan committed:
Change to check for null instead of undefined

jamesduncan committed:
add additional check on rowData

jamesduncan committed:
only use ?.

Achoobert committed:
Merge pull request #155 from digi-serve/is/FixprocessMultiUser-core

treat multi user like many-many

jamesduncan committed:
Merge pull request #154 from digi-serve/hotfix_unset_cursor_formatnumber

Handle null rowData after unset cusor

Achoobert committed:
Revert "Merge remote-tracking branch 'origin/v2' into is/fixSDCurl"

This reverts commit dc9319f0ccfce4eca7d82af451d16f8860943c3f.

Achoobert committed:
Merge remote-tracking branch 'origin/v2' into is/fixSDCurl
